web/src/main/java/no/nav/ung/sak/web/server/abac/PdpRequestMapper.java

```diff
@@ -23,20 +22,20 @@ public static SaksinformasjonTilgangskontrollInputDto map(PdpRequest pdpRequest)
     }
 
     public static OperasjonDto operasjon(PdpRequest pdpRequest) {
-        ResourceType resource = resourceTypeFraKode(pdpRequest.getString(no.nav.ung.sak.tilgangskontroll.api.AbacAttributter.RESOURCE_FELLES_RESOURCE_TYPE));
-        return new OperasjonDto(resource, actionFraKode(pdpRequest.getString(no.nav.ung.sak.tilgangskontroll.api.AbacAttributter.XACML_1_0_ACTION_ACTION_ID)));
+        ResourceType resource = resourceTypeFraKode(pdpRequest.getString(AbacAttributter.RESOURCE_FELLES_RESOURCE_TYPE));
+        return new OperasjonDto(resource, actionFraKode(pdpRequest.getString(AbacAttributter.XACML_1_0_ACTION_ACTION_ID)));
     }
 
     public static SaksinformasjonDto saksinformasjon(PdpRequest pdpRequest) {
         return new SaksinformasjonDto(
-            pdpRequest.getString(no.nav.ung.sak.tilgangskontroll.api.AbacAttributter.RESOURCE_K9_SAK_ANSVARLIG_SAKSBEHANDLER),
+            pdpRequest.getString(AbacAttributter.RESOURCE_K9_SAK_ANSVARLIG_SAKSBEHANDLER),
```

Comment:
Burde vi ha egen AbacAttributt for ung-sak ressurs?

Reply:
Har endret til ung nå, brukes for sporingslogg. For tilgangskontroll er det ingen endring da den mappes om før den sendes ut fra ung-sak.